src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,60 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY_CONS = "supply";
+    private static final String BUY_CONS = "buy";
+    private static final String RESULT_CONS = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int[] totals = readFile(fromFileName);
+        int totalSupply = totals[0];
+        int totalBuy = totals[1];
+        writeToFile(toFileName, totalSupply, totalBuy);
+    }
```

Comment:
The `getStatistic` method is doing too much by both reading and writing to a file. This violates the principle of having methods that are responsible for one task. Consider splitting the logic into separate methods to handle reading and processing the data, and another for writing the processed data to a file.